apps/smtp/src/modules/smtp/configuration/smtp-metadata-manager.ts

```diff
@@ -1,42 +1,26 @@
 import { SettingsManager } from "@saleor/app-sdk/settings-manager";
 import { SmtpConfig } from "./smtp-config-schema";
-import { SmtpPrivateMetadataManagerV2 } from "./smtp-metadata-manager-v2";
-import { smtpConfigMigrationV1ToV2 } from "./migrations/smtp-config-migration-v1-to-v2";
-import { createLogger } from "../../../logger";
 
-const logger = createLogger("SmtpPrivateMetadataManager");
-
-export class SmtpPrivateMetadataManager {
-  private metadataKey = "smtp-config-v2";
+export class SmtpMetadataManager {
+  private metadataKey = "smtp-config";
 
   constructor(
     private metadataManager: SettingsManager,
     private saleorApiUrl: string,
   ) {}
 
-  async getConfig() {
-    logger.debug("Fetching config in the current version");
-
-    const currentVersionManager = new SmtpPrivateMetadataManagerV2(
-      this.metadataManager,
-      this.saleorApiUrl,
-    );
-
-    const currentVersionConfig = await currentVersionManager.getConfig();
-
-    if (currentVersionConfig) {
-      // We have the current version, no need to migrate so we can return it
-      return currentVersionConfig;
-    }
-
-    logger.debug("No config in the current version, trying to migrate from v1");
-    // TODO: MIGRATION CODE FROM CONFIG VERSION V1. REMOVE AFTER MIGRATION
-    const migratedSchema = await smtpConfigMigrationV1ToV2({
-      saleorApiUrl: this.saleorApiUrl,
-      settingsManager: this.metadataManager,
+  getConfig(): Promise<SmtpConfig | undefined> {
+    return this.metadataManager.get(this.metadataKey, this.saleorApiUrl).then((data) => {
+      if (!data) {
+        return data;
+      }
+
+      try {
+        return JSON.parse(data);
+      } catch (e) {
+        throw new Error("Invalid metadata value, can't be parsed");
```

Comment:
question: why did we remove logger from this class? is this error caught and logged somewhere else?

Reply:
Logs were removed because they were related to migration logic that was removed

I will work on logs and error handling in further PRs and complete overhaul them